js/bybit.js

```diff
@@ -1947,142 +1866,103 @@ module.exports = class bybit extends Exchange {
-        //             "v": "0.907276"
-        //             }
-        //         ]
-        //         },
-        //         "retExtInfo": {},
-        //         "time": 1659430400353
-        //     }
-        //
-        const result = this.safeValue (response, 'result', {});
-        const ohlcvs = this.safeValue (result, 'list', []);
-        return this.parseOHLCVs (ohlcvs, market, timeframe, since, limit);
-    }
-
-    async fetchDerivativesOHLCV (symbol, timeframe = '1m', since = undefined, limit = undefined, params = {}) {
+    async fetchOHLCV (symbol, timeframe = '1m', since = undefined, limit = undefined, params = {}) {
+        /**
+         * @method
+         * @name bybit#fetchOHLCV
+         * @description fetches historical candlestick data containing the open, high, low, and close price, and the volume of a market
+         * @param {string} symbol unified symbol of the market to fetch OHLCV data for
+         * @param {string} timeframe the length of time each candle represents
+         * @param {int|undefined} since timestamp in ms of the earliest candle to fetch
+         * @param {int|undefined} limit the maximum amount of candles to fetch
+         * @param {object} params extra parameters specific to the bybit api endpoint
+         * @returns {[[int]]} A list of candles ordered as timestamp, open, high, low, close, volume
+         */
+        this.checkRequiredSymbol ('fetchOHLCV', symbol);
         await this.loadMarkets ();
         const market = this.market (symbol);
-        if (market['option']) {
-            throw new NotSupported (this.id + ' fetchOHLCV() is not supported for option markets');
-        }
         const request = {
             'symbol': market['id'],
         };
         const duration = this.parseTimeframe (timeframe);
-        const now = this.milliseconds ();
         if (limit === undefined) {
             limit = 200; // default is 200 when requested with `since`
-        } else {
-            request['limit'] = limit;
         }
         if (since === undefined) {
-            since = now - (limit * duration * 1000);
+            request['start'] = since;
         }
-        // end is required parameter
-        let end = this.safeInteger (params, 'end');
-        if (end === undefined) {
-            end = this.sum (since, limit * duration * 1000);
+        if (limit !== undefined) {
+            request['limit'] = limit; // max 1000, default 1000
         }
-        if (market['linear']) {
-            request['category'] = 'linear';
-        } else if (market['inverse']) {
-            request['category'] = 'inverse';
+        request['interval'] = duration;
+        let method = undefined;
+        if (market['spot']) {
+            request['category'] = 'spot';
+            method = 'publicGetV5MarketKline';
+        } else {
+            const price = this.safeString (params, 'price');
+            params = this.omit (params, 'price');
+            const methods = {
```

Comment:
@sc0Vu I know you didn't add them but you can remove them because they are already implemented in the base exchange 
![image](https://user-images.githubusercontent.com/43336371/219652039-da766262-3a96-49a9-968e-09df059947d3.png)